cmd/ctr/run.go

```diff
@@ -36,6 +36,15 @@ var runCommand = cli.Command{
 		if id == "" {
 			return fmt.Errorf("container id must be provided")
 		}
+
+		bundle, err := filepath.Abs(context.String("bundle"))
```

Comment:
You may as well rename `validateBundlePath` to `getBundlePath` pass it `context.String("bundle")` and let it do both the call to `filepath.Abs` and `os.Stat`